slicer/slicerrc.py

```diff
@@ -0,0 +1,217 @@
+    seg.GetSegmentation().AddEmptySegment("3", "Left lateral ventricle")
+    seg.GetSegmentation().AddEmptySegment("4", "Right lateral ventricle")
+
+
+def save_all(filepath, annotations_dir=ANNOTATIONS_DIR):
+    nodes = getNodesByClass('vtkMRMLScalarVolumeNode')
+    assert len(nodes) == 1
+    node = nodes[0]
+    studyUID = os.path.basename(filepath)[:-len('.nii.gz')]
+    dir_path = os.path.join(annotations_dir, studyUID)
+    segnodes = getNodesByClass('vtkMRMLSegmentationNode')
+    print(segnodes)
+    if len(segnodes) == 1:
+        if not os.path.exists(dir_path):
+            print('{} does not exists, Creating directory.'.format(dir_path))
+            os.mkdir(dir_path)
+        segnode = segnodes[0]
+        slicer.util.saveNode(segnode, os.path.join(dir_path, studyUID + '.seg.nrrd'))
+        slicer.util.saveNode(node, os.path.join(dir_path, studyUID + '.nrrd'))
+    markupnodes = getNodesByClass('vtkMRMLMarkupsNode')
+    markuplinenodes = [node for node in markupnodes if 'MarkupsLine' in str(node)]
+    markupcurvenodes = [node for node in markupnodes if 'MarkupsCurve' in str(node)]
+    for i, markupnode in enumerate(markuplinenodes):
+        slicer.util.saveNode(markupnode, os.path.join(dir_path, f'L_{str(i)}.mrk.json'))
+    for i, markupnode in enumerate(markupcurvenodes):
+        slicer.util.saveNode(markupnode, os.path.join(dir_path, f'C_{str(i)}.mrk.json'))
+    save_resampled_points(markupcurvenodes, dir_path)
+    save_as_ijk(markupcurvenodes, dir_path, 'C')
+    save_as_ijk(markuplinenodes, dir_path, 'L')
+    print('Saving all segmentations to {}'.format(dir_path))
+
+def get_ijk_point(pt):
+    volumeID = 'vtkMRMLScalarVolumeNode1'
+    coordsXYZ = [0, 0, 0]
+    sliceNode = getNode('vtkMRMLSliceNodeRed')
+    appLogic = slicer.app.applicationLogic()
+    sliceLogic = appLogic.GetSliceLogic(sliceNode)
+    layerLogic = sliceLogic.GetBackgroundLayer()
+    slicer.vtkMRMLAbstractSliceViewDisplayableManager.ConvertRASToXYZ(sliceNode,pt,coordsXYZ)
+    xytoIJK = layerLogic.GetXYToIJKTransform()
+    coordsIJK = xytoIJK.TransformDoublePoint(coordsXYZ)
+    coordsIJK = list(map(int, coordsIJK))
+    return coordsIJK
+
+def save_as_ijk(markupnodes, dir_path, prefix):
+    for i, node in enumerate(markupnodes):
+        try:
+            currentPoints = node.GetCurvePointsWorld()
+            all_points = {'0': [], '1': [], '2': []}
+            for controlpoint in range(0, currentPoints.GetNumberOfPoints()):
+                pt = [0, 0, 0]
+                currentPoints.GetPoint(controlpoint, pt)
+                pt_ijk = get_ijk_point(pt)
+                for i in range(3):
+                    all_points[str(i)].append(pt_ijk[i])
+        
+            with open(os.path.join(dir_path, f'ijk_{prefix}_{str(i)}.json'),'w') as fp:
+                json.dump(all_points, fp)
+        except AttributeError:
+            continue
+
+def save_resampled_points(markupcurvenodes, dir_path):
```

Comment:
can you add documentation to most of the functions here? I think we should rename some functions because they are not very readable.